README.md

```diff
@@ -2,6 +2,13 @@
 
 A Django-based website for http://globaltrademotors.com
 
+# Added travis-ci status badge to README
+
+![alt text](https://travis-ci.org/travis-ci/travis-web.svg?branch=daily-cron)
+<!---
+# Added a Badge with Username and Repo-name
+![alt text](https://travis-ci.org/{{sitture}}/{{trade-motors}}.png) -->
```

Comment:
Rather than this, if you go to https://travis-ci.org/sitture/trade-motors, then click the build badge and select Markdown from the dropdown, you can then copy and paste this more-standard code:
```md
[![Build Status](https://travis-ci.org/sitture/trade-motors.svg?branch=master)](https://travis-ci.org/sitture/trade-motors)
```

Reply:
Sure, will update.